rest/resource-server/src/docs/asciidoc/projects.adoc

```diff
@@ -650,3 +650,17 @@ include::{snippets}/should_document_import_sbom/curl-request.adoc[]
 
 ===== Example response
 include::{snippets}/should_document_import_sbom/http-response.adoc[]
+
+[[resources-project-getprojectcount]]
+==== Get a project count of user
+
+A `GET` request will get a project count.
```

Comment:
Change the message to, A `GET` request will get the project count of a user or something like that.
Same for line no. 655.